### DISCUSSION TITLE
I am a beginner, why is there a warning? I have no problem using it, but there will be a warning.

### DISCUSSION THREAD

**Discussion by bbhxwl** [sqlalchemy/sqlalchemy#11630] [Usage Questions]
<img width="828" alt="image" src="https://github.com/user-attachments/assets/906a3261-ccfe-44d4-986e-07acabf970a9">

**Comment by gordthompson** [comment]
Are you using PyCharm?

**Reply by bbhxwl** [reply]
yes 
![image](https://github.com/user-attachments/assets/5ca5e710-53f2-43f7-aff3-1588d28893a9)

**Comment by gordthompson** [comment]
Duplicate of #11070
